<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OUCmap</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Shortestpath/timg.jpg</normaloff>../Shortestpath/timg.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="btn_cailiao">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>材料学院</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_dongcao">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>215</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>东操</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_beichan">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>135</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>北餐</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_sijiao">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>四教</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_wujiao">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>225</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>五教</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_beicao">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>45</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>北操</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_diyuan">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>495</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>地球学院</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_gongyuan">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>325</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>工院</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_huayuan">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>425</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>化院</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_xinbei">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>425</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>信北</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_xinnan">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>495</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>信南</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_houqin">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>735</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>后勤宿舍</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_wuziding">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>365</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>五子顶</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_wenxin">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>805</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>文新</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_haiyang">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>365</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>海洋学院</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_haike">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>175</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>海科楼</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_tiyuguan">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>105</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>体育馆</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_liujiao">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>295</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>六教</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_tushuguan">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>645</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>图书馆</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_xinzilou">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>行知楼</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_shuyuan">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>755</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="text">
     <string>数院</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_zhengfa">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>685</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>政法楼</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>166</y>
      <width>291</width>
      <height>661</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_2" stretch="0,0,0,1,2,2,0">
     <item>
      <widget class="QPushButton" name="btn_dijkstra">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>2</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Dijkstra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_floyd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>2</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>SimSun-ExtB</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Floyd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clean">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clean</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>18</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>   start：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_start">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Arial&quot;;
color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>   end：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_over">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Arial&quot;;
color: rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_display">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Arial&quot;;
color: rgb(105, 158, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>249</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>使用方法：
先依次点击起点和终点
再选择下方的算法

点击Clean重置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
